composer.json

```diff
@@ -51,7 +51,7 @@
         "ext-pdo_sqlite": "*",
         "payum/omnipay-v3-bridge": "^1.0",
         "omnipay/dummy": "^3.0",
-        "omnipay/common": "^3.0",
+        "omnipay/common": "3.0.x-dev",
```

Comment:
There is no tagged version for `omnipay/common` yet that supports PHP 8, but support is added in the latest dev branch. So without this change, dependencies can't be reliably updated. Since this is just a dev dependency, it should be fine to keep it as an unbound constraint to allow the tests to run, which can then be updated as soon as there is a new tagged version

Reply:
This has been moved to Travis rather to only install dev dependencies in the CI